sc3nb/sc.py

```diff
@@ -351,19 +354,24 @@ def exit(self):
             if self.server:
                 self.__s_quit()
         self.osc.exit()
+        print("Killing sclang subprocess")
         self.scp.kill()
+        self.scp.wait()
+        print(f"Done. ")
 
     def boot_with_blip(self):
         """Boots SuperCollider server with audio feedback
         """
 
-        print('Booting server...')
+        print('Booting scsynth...')
```

Comment:
actually, scsynth is the process, which features the server. As it is also said to boot the server and not booting scsynth in SC3, the wording is maybe misleading, I suggest 'Booting sc3 server...'